docs/API/Elements.md

```diff
@@ -41,7 +43,7 @@ function enter() {
 
```

Comment:
Looks like there needs to be a `<br>` between the "Any created asset..." section and the "`This` is a helpful keyword...," but otherwise looks good to me!